examples/java/pom.xml

```diff
@@ -283,6 +287,34 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.beam</groupId>
```

Comment:
examples/common/ExampleUtils has dependency on beam-runners-google-cloud-dataflow-java. It can't be set in runtime scope.